backend/src/main/java/org/eclipse/tractusx/puris/backend/common/edc/logic/util/EDCRequestBodyBuilder.java

```diff
@@ -167,7 +167,16 @@ public JsonNode buildTransferRequestBody(String transferId,
         transferNode.set("transferType", transferTypeNode);
         transferNode.put("managedResources", false);
         propertiesNode = MAPPER.createObjectNode();
+        // TODO: reminder for EDC 0.5.x: receiverHttpEndpoint - auth key and code still not possible
```

Comment:
Those TODOs should be extracted into the issue board of the project to keep track of it.